CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing code
+
+The fundamental code standard we use
+[StandardJS](http://standardjs.com/rules.html), the rules being
+described in the aforementioned link. We highly recommend you stick to
```

Comment:
Actually, we plan on putting linters in the CI process so all PRs that don't stick to the standardjs style get a CI fail.